applications/openshift/api-server/api_server_anonymous_auth/rule.yml

```diff
@@ -7,9 +7,9 @@ title: 'Disable Anonymous Authentication to the API Server'
 description: |-
     By default, anonymous access to the OpenShift API is enabled. This
     configuration check ensures that anonymous requests to the OpenShift
-    API server are disabled. Edit the API server pod specification file
+    API server are disabled. 
 {{%- if product == "ocp4" %}}
-    <tt>/etc/kubernetes/static-pod-resources/configmaps/config/config.yaml</tt> on the master node(s)
+    Edit the <tt>openshift-kube-apiserver</tt> configmap on the master node(s)
```

Comment:
This configmap is the current running config, and whenever you try to edit this configmap manually the kube-apiserver-operator will just override it. It pieces it together through other APIs and here not a lot of it exposed for configuration purposes. It might be that most of these would have to be implemented as an unsupportedConfigOverrides (see `oc explain kubeapiserver.spec` ).

Reply:
@redhatrises let's comment out this explanation, and eventually switch it to refer to the instructions @mrogers950 is giving.